scripts/setup-raspbian.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+# setup raspbian / raspberry pi configuration files
+# /boot/cmdline.txt and
+# /boot/config.txt
+
+. $COMPANION_DIR/scripts/bash-helpers.sh
+
+# Disable camera LED
+run_step sudo sed -i '\%disable_camera_led=1%d' /boot/config.txt
+run_step sudo sed -i '$a disable_camera_led=1' /boot/config.txt
+
+# Enable RPi camera interface
+run_step sudo sed -i '\%start_x=%d' /boot/config.txt
+run_step sudo sed -i '\%gpu_mem=%d' /boot/config.txt
+run_step sudo sed -i '$a start_x=1' /boot/config.txt
+run_step sudo sed -i '$a gpu_mem=128' /boot/config.txt
+
+#Delete ip address if present from /boot/cmdline.txt
+# e.g. sed command removes any ip address with any combination of digits [0-9] between decimal points
+sudo sed -i -e 's/\s*ip=[0-9]*\.[0-9]*\.[0-9]*\.[0-9]*//' /uboot/cmdline.txt || sudo sed -i -e 's/\s*ip=[0-9]*\.[0-9]*\.[0-9]*\.[0-9]*//' /boot/cmdline.txt
+
+# remove any line containing 'enable_uart=' from /boot/config.txt
+run_step sudo sed -i '/enable_uart=/d' /boot/config.txt
```

Comment:
The uart console is not working..

Reply:
yes, that's the idea. We should remove uart console to use ping or any other serial sensor.

Reply:
thank you <3 I see it now.